src/components/App/App.jsx

```diff
@@ -11,20 +13,8 @@ const App = () => {
 
   return (
```

Comment:
This component needs to set the heights and widths for Navbar and HomePage. and make sure they size in mobile and nothing overflows once HomePage starts growing